TicketDesk/ngWebClient-StandAlone/src/app/single-ticket-view/single-ticket-view.component.ts

```diff
@@ -3,32 +3,36 @@ import { ActivityLogComponent } from '../activity-log/activity-log.component';
 import { ContactInfoComponent } from '../contact-info/contact-info.component';
 import { TicketActionEntryComponent } from '../ticket-action-entry/ticket-action-entry.component';
 import { SingleTicketService } from '../services/single-ticket.service';
-import { Ticket } from '../models/data';
-import {Router, ActivatedRoute, Params} from '@angular/router';
-
+import { Ticket } from '../models/ticket';
+import { Router, ActivatedRoute, Params } from '@angular/router';
 
 @Component({
   selector: 'app-single-ticket-view',
   templateUrl: './single-ticket-view.component.html',
   styleUrls: ['./single-ticket-view.component.css']
 })
 export class SingleTicketViewComponent implements OnInit {
-
   ticket: Ticket = null;
   ticketId: number = null;
   ticketActionPermissions = 0;
   public isCollapsed = true;
-
+  ticketStatus: string;
   constructor(private singleTicketService: SingleTicketService,
     private activatedRoute: ActivatedRoute) {
     this.activatedRoute.params.subscribe(params => {
       this.ticketId = Number(params['ticketID']);
     });
   }
 
+  ticketIsOpen(): boolean {
+    const status = this.ticketStatus.toLowerCase();
+    return status.indexOf('active') !== -1
```

Comment:
is this how people do this??????????????? it looks very intentional, but it's making me real uncomfortable

Reply:
Agreed, why not use something like .includes() instead? Index seems unnecessary amount of work.